src/components/src/modals/cloud-tile.tsx

```diff
@@ -78,7 +79,7 @@ const StyledUserName = styled.div`
 `;
 
 interface OnClickProps {
-  onClick?: React.MouseEventHandler<HTMLDivElement>;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
since we are passing props to a button makes more sense to target HTMLButtonElement